maps4fs/generator/texture.py

```diff
@@ -242,18 +245,33 @@ def layers(self, layers: list[Layer]) -> None:
     # pylint: disable=no-member
     def draw(self) -> None:
         """Iterates over layers and fills them with polygons from OSM data."""
-        for layer in self.layers:
+        layers = sorted(self.layers, key=lambda _layer: _layer.priority)
+        cumulative_image = None
+        for layer in layers:
             if not layer.tags:
                 self.logger.debug("Layer %s has no tags, there's nothing to draw.", layer.name)
                 continue
+            if layer.name == "grass":
```

Comment:
Add a 0 priority for the "base" layer. So we won't compare some magic strings here and will only check if the layer priority is 0. It also will be easier to update and/or maintain.

Reply:
E.g.
```python
if layer.priority == 0:
```